src/pybind/mgr/orchestrator/module.py

```diff
@@ -213,6 +213,23 @@ class DaemonAction(enum.Enum):
     rotate_key = 'rotate-key'
 
 
+class NonCephImageServiceTypes(enum.Enum):
+    prometheus = 'prometheus'
+    loki = 'loki'
+    promtail = 'promtail'
+    node_exporter = 'node-exporter'
+    alertmanager = 'alertmanager'
+    grafana = 'grafana'
+    nvmeof = 'nvmeof'
+    snmp_gateway = 'snmp-gateway'
+    elasticsearch = 'elasticsearch'
+    jaeger_collector = 'jaeger-collector'
+    jaeger_query = 'jaeger-query'
+    jaeger_agent = 'jaeger-agent'
+    samba = 'smb'
+    oauth2_proxy = 'oauth2-proxy'
```

Comment:
I'd prefer if we could continue to have this sort of thing be part of that new python-common/ceph/cephadm/images.py. Part of the point of that was so that if we update the name of an image setting, add a new image etc. we only need to update one place. This is adding a separate place to worry about that could be easily missed.